docs/whatsnew/5.3.rst

```diff
@@ -111,32 +111,37 @@ It is now possible to read and write compressed FITS files that make use of the
 uncompressed tiles by specifying ``compression_type='NOCOMPRESS'`` in
 :class:`~astropy.io.fits.CompImageHDU`.
 
-.. _whatsnew-5.3-unit-formats-inline:
+.. _whatsnew-5.3-unit-formats-fraction-inline:
 
-New "inline" option for ``"unicode"``, ``"console"``, and ``"latex"`` unit formats
-==================================================================================
+New "fraction" and "inline" options for representing units as strings
+=====================================================================
 
-A new formatting option for unit is added to switch between inline and
-multiline prettyprinting of quantities::
+New formatting options are added to switch between using fractions or
+using negative powers directly, and between inline and multiline
+prettyprinting of quantities::
 
 	>>> import astropy.units as u
 	>>> unit = u.Unit("erg / (s cm2)")
 	>>> print(unit.to_string('console'))
 	erg s^-1 cm^-2
-	>>> print(unit.to_string('console', inline=False))
+	>>> print(unit.to_string('console', fraction=True))
          erg
         ------
         s cm^2
+	>>> print(unit.to_string('console', fraction=True, inline=True))
+        erg / (s cm^2)
 	>>> print(unit.to_string('unicode'))
 	erg s⁻¹ cm⁻²
-	>>> print(unit.to_string('unicode', inline=False))
+	>>> print(unit.to_string('unicode', fraction=True))
          erg
         ─────
         s cm²
+	>>> print(unit.to_string('unicode', fraction=True, inline=True))
+        erg / (s cm²)
 
 Note that the ``"console"`` and ``"unicode"`` formats now use
-``inline=False`` as default, while for ``"latex"`` the default remains
-``inline=True``, for an unchanged experience with IPython notebook.
+``fraction=False`` as default, while for ``"latex"`` the default remains
+``fraction=True, inline=False``, for an unchanged experience with IPython notebook.
```

Comment:
Is this outdateted?

Reply:
Yes, oops!  Also forgot the `whatsnew` which is causing all the failures. Sigh!